testsuite/features/core/srv_create_activationkey.feature

```diff
@@ -116,8 +116,3 @@ Feature: Create activation keys
     And I check "OS Image Build Host"
     And I click on "Create Activation Key"
     Then I should see a "Activation key Build host Key x86_64 has been created" text
-    And I should see a "Details" link
-    And I should see a "Packages" link
-    And I should see a "Configuration" link in the content area
-    And I should see a "Groups" link
-    And I should see a "Activated Systems" link
```

Comment:
Are we losing coverage here? Or having these fields on the WebUI is covered somewhere else?

Reply:
Explained in the PR description: it's duplicated lines, probably by copy and paste accident.

So no we don't lose coverage. Same lines are above in same file.